cmdLine/src/main/scala/topgun/cmdline/FileParser.scala

```diff
@@ -154,13 +201,25 @@ class FileParser(file: File, cmdLine: JfrParseCommandLine, totals: Totals, confi
 
   private def readFrames(event: RecordedEvent): List[CallSite] = {
     val stack = event.getStackTrace
-
     if (stack eq null) List() else {
       stack.getFrames.iterator.asScala.map {
         frame: RecordedFrame =>
           val method = frame.getMethod
-          CallSite(method.getType.getName, method.getName, method.getName, method.getDescriptor, frame.getLineNumber)
-      }.distinct.takeWhile { f => !f.isIgnorableTopFrame }.toList
+          if(method ne null) {
+            val splitIndex = method.getType.getName.lastIndexOf(".");
+            CallSite(
+              method.getType.getName.substring(0,splitIndex).intern(),
+              method.getType.getName.substring(splitIndex+1).intern(),
+              method.getName.intern(),
+              method.getDescriptor.intern(),
+              frame.getLineNumber,
+              classLoaderKey
+            )
+          } else {
+            CallSite(new String("NoMethodInFrame_").intern, new String("NoMethodInFrame_").intern, new String("NoMethodInFrame_").intern, new String("NoMethodInFrame_").intern, frame.getLineNumber,classLoaderKey)
```

Comment:
dont use `new String("Something").intern` constants are interned anyway, so that s the same as `"Something"`
It just creates a String and then throws it away when it interns